scripts/pyproject.toml

```diff
@@ -13,10 +13,10 @@ pygeos = "^0.12.0"
 pandas = "^1.4.2"
 click = "^8.1.2"
 
-[tool.poetry.dev-dependencies]
-ipykernel = "^6.13.0"
-isort = "^5.10.1"
-black = "^22.3.0"
+[tool.poetry.group.dev.dependencies]
```

Comment:
Oh interesting; this must be a change with poetry 1.2 from `dev-dependencies` to `group.dev`

Reply:
Yeah, if we need to support poetry < 1.2, I could move these back.  `poetry add --dev ...` (which has been replaced with `poetry add --group dev`) now writes to this group instead of `dev-dependencies`.  See https://python-poetry.org/docs/managing-dependencies/#dependency-groups for some additional detail.

Reply:
I'm fine with keeping >=1.2